schema/instance.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:generate stringer -type=Type -linecomment
+
 package schema
```

Comment:
As mentioned in https://github.com/ConduitIO/conduit-connector-protocol/pull/96#discussion_r1657571765, I think it would be good to create a proto file here just for a schema object that can be reused in processors and connectors.